CodeReview/src/b/Textdatei b

```diff
@@ -0,0 +1,4 @@
+Auch hier ist wie bereits bei a die Id als public definiert und zusätzlich kann man hier die Id noch selber verändern.
+Ich nehme an der var_dump ist nur zum anzeigen gedacht. Der hätte auch nichts verloren in einem script.
+
+Die Id sollte wenn möglich einmalig und zufällig sein und man sollte sie nicht mit einer Set Methode verändern können.
```

Comment:
das kommt wohl auf den konkreten case in der applikation an. aber grundsätzlich hast du schon recht.